lib/Math/Sequences/Integer.pm6

```diff
@@ -509,7 +509,14 @@ our @A000609 is export = 1, &NOSEQ ... *;
 # A000670 / preferential arrangements
 our @A000670 is export = 1, &NOSEQ ... *;
 # A000688 / abelian groups
-our @A000688 is export = 1, &NOSEQ ... *;
+our @A000688 is export our @A000688 = (1..*).map: {
+    # @A000041 NYA. Hardcoded list, will fail at term 2 ** 173526
```

Comment:
NYA?

![nya](https://image.myanimelist.net/ui/3jlDHhq19EFYUkjjdMDkyN7L1Lll3MMHk52iVZlorbl1-V625LAXFPxEs0LRY4pkFWdFLYvvQ-N48FLnEBACG6a2dC9mir2Zo7cN62f1C2XYNY1zD__ppHfc8Ip0BiIQ)

Reply:
I was very confused for a short while until I figured out what you were referring to. :-)